admin/helpers/check-form-presenter.js

```diff
@@ -54,4 +54,42 @@ checkFormPresenter.getFlashMessageData = (uploadData) => {
   return highlightData
 }
 
+/**
+ * Format check window list data for the assign forms to check windows view
+ * @param {Array} checkWindows
+ * @returns {Array}
+ */
+checkFormPresenter.getPresentationCheckWindowListData = (checkWindows) => {
+  const checkWindowData = []
+  checkWindows.forEach(cw => {
+    checkWindowData.push({
+      name: cw.name,
+      urlSlug: cw.urlSlug,
+      familiarisationCheckStartDate: cw.familiarisationCheckStartDate.format('D MMMM'),
```

Comment:
Why aren't these calls using the dateService?

Reply:
@jon-shipley updated based on our discussion